RELEASE.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## 1.2.0
```

Comment:
I think we should get in the habit of putting periods at the end of each phrase or sentence in the RELEASE doc.

Reply:
I don't see a line that outlines the Legend. Can this be added?